contractsV5/KyberNetwork.sol

```diff
@@ -266,20 +266,27 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         returns (uint expectedRate, uint worstRate)
     {
         if (src == dest) return (0, 0);
+        uint permHintGetRate = 1 << 255; //for backwards compatibility
```

Comment:
I think constant is still preferred
otherwise this is calculated per function call.
constant is calculated on compilation time.
but I am not sure about this.
Should check the assembly...
or check gas consumption. 
or see documentation.
which ever is easier

Reply:
20 gas cheaper with global constant